public/src/components/teacher/Results.tsx

```diff
@@ -125,7 +125,7 @@ export class Results extends React.Component<IResultsProp, IResultsState> {
     }
 
     private generateUserRepoLink(name: string, userName: string): JSX.Element {
-        return <a href={generateLabRepoLink(this.props.courseURL, userName)} target="_blank">{ name }</a>;
```

Comment:
Similar to `groupRepoLink()` this function should also be moved to `componentHandler.tsx` as `userRepoLink()`.
Also do a search for `generateGitLink` to see if more of these can be replaced with `userRepoLink()`

Reply:
Done.